clients/ctl/admin-ui/src/features/dataset/DataCategoryInput.tsx

```diff
@@ -33,11 +36,23 @@ const DataCategoryInput = ({
       <Stack>
         <Box display="flex" alignItems="center">
           <Box mr="2" width="100%">
-            <DataCategoryDropdown
-              dataCategories={dataCategories}
-              checked={checked}
-              onChecked={onChecked}
-            />
+            {features.plus ? (
+              <ClassifiedDataCategoryDropdown
+                dataCategories={dataCategories}
+                // TODO: make this real
+                mostLikelyCategories={dataCategories
+                  .slice(0, 5)
+                  .map((d) => ({ ...d, confidence: 97 }))}
+                checked={checked}
+                onChecked={onChecked}
+              />
+            ) : (
+              <DataCategoryDropdown
+                dataCategories={dataCategories}
+                checked={checked}
+                onChecked={onChecked}
+              />
+            )}
```

Comment:
The two separate stacks of categories is definitely funky. I was imagining that the new `select` would show all of the `checked` categories, whether or not they were from the classifier (but it would highlight the classifier's confidence). That would make the `TaxonomyEntityTag` below unnecessary.

Reply:
ahh I see what you mean. I'll try it out!